multicluster/controllers/multicluster/node_controller.go

```diff
@@ -37,18 +38,22 @@ import (
 
 type (
 	// NodeReconciler is for member cluster only.
-	// It will create a Gateway object if a Node has an annotation `multicluster.antrea.io/gateway:true`
-	// and update corresponding Gateway if any subnets changes.
 	NodeReconciler struct {
 		client.Client
-		Scheme     *runtime.Scheme
-		namespace  string
-		precedence mcsv1alpha1.Precedence
+		Scheme            *runtime.Scheme
+		namespace         string
+		precedence        mcsv1alpha1.Precedence
+		gatewayCandidates sets.String
```

Comment:
I feel it is simpler (also more efficient) to use a map from what I saw in code, than sets.String.

Reply:
Done